packages/javascript-mutator/src/mutators/ArrayDeclarationMutator.ts

```diff
@@ -8,19 +8,21 @@ import { NodeMutator } from './NodeMutator';
 export default class ArrayDeclarationMutator implements NodeMutator {
   public name = 'ArrayDeclaration';
 
-  public mutate(node: types.Node, copy: <T extends types.Node>(obj: T, deep?: boolean) => T): types.Node[] {
-    const nodes: types.Node[] = [];
-
-    if (types.isArrayExpression(node)) {
-      const mutatedNode = copy(node);
-      mutatedNode.elements = node.elements.length ? [] : [types.stringLiteral('Stryker was here')];
-      nodes.push(mutatedNode);
-    } else if ((types.isCallExpression(node) || types.isNewExpression(node)) && types.isIdentifier(node.callee) && node.callee.name === 'Array') {
-      const mutatedNode = copy(node);
-      mutatedNode.arguments = node.arguments.length ? [] : [types.arrayExpression()];
-      nodes.push(mutatedNode);
-    }
-
-    return nodes;
+  public mutate(node: types.Node): Array<[types.Node, types.Node | { raw: string }]> {
+    return types.isArrayExpression(node)
```

Comment:
I really don't like this shorted if statement. It can be too unclear to read. Consider writing it as normal `if/else`

Reply:
I am fine to change all mutators back to normal if/else statements, would like to see PR #1903 reviewed and merged first.

Reply:
Should be fixed now